angular-workspace/package.json

```diff
@@ -54,7 +54,7 @@
     "rxjs": "~7.8.1",
     "storybook": "7.6.20",
     "tslib": "^2.8.1",
-    "typescript": "~5.5.4",
+    "typescript": "~5.6.3",
```

Comment:
Er is een warning in de build log:

#23 47.77 ➤ YN0060: │ typescript is listed by your project with version 5.6.3 (p2101f), which doesn't satisfy what @angular-devkit/build-angular and other dependencies request (>=5.4.0 <5.6.0-0).

Daarom heb ik tot nog toe de versie op ~5.5.4 gehouden

Reply:
done